src/tv2_afvd_showstyle/getRundown.ts

```diff
@@ -579,61 +578,45 @@ function getBaseline(context: ExtendedShowStyleContext<GalleryBlueprintConfig>):
 						}
 				  })
 				: undefined,
-			literal<TSR.TimelineObjAtemSsrcProps>({
-				id: '',
-				enable: { while: '1' },
-				priority: 0,
-				layer: SwitcherDveLLayer.Dve,
-				content: {
-					deviceType: TSR.DeviceType.ATEM,
-					type: TSR.TimelineContentTypeAtem.SSRCPROPS,
-					ssrcProps: {
-						artFillSource: context.config.studio.SwitcherSource.SplitArtF,
-						artCutSource: context.config.studio.SwitcherSource.SplitArtK,
-						artOption: 1,
-						artPreMultiplied: true
-					}
-				}
-			}),
-			literal<TSR.TimelineObjAtemSsrc>({
-				id: '',
+			...context.videoSwitcher.getDveTimelineObjects({
 				enable: { while: '1' },
-				priority: 0,
-				layer: SwitcherDveLLayer.DveBoxes,
 				content: {
-					deviceType: TSR.DeviceType.ATEM,
-					type: TSR.TimelineContentTypeAtem.SSRC,
-					ssrc: {
-						boxes: [
-							{
-								// left
-								enabled: true,
-								source: AtemSourceIndex.Bars,
-								size: 580,
-								x: -800,
-								y: 50,
-								cropped: true,
-								cropRight: 2000
-							},
-							{
-								// right
-								enabled: true,
-								source: AtemSourceIndex.Bars,
-								size: 580,
-								x: 800,
-								y: 50
-								// note: this sits behind box1, so don't crop it to ensure there is no gap between
-							},
-							{
-								// box 3
-								enabled: false
-							},
-							{
-								// box 4
-								enabled: false
-							}
-						]
-					}
+					boxes: [
+						{
+							// left
+							enabled: true,
+							source: AtemSourceIndex.Bars,
```

Comment:
This is not a valid source number right now. I think it went unnoticed because the DveProps interface still had its todos (`any`)

Reply:
What should it be instead?